src/server/v2.0/handler/auditlog.go

```diff
@@ -49,17 +49,17 @@ type auditlogAPI struct {
 	projectCtl  project.Controller
 }
 
-func (a *auditlogAPI) ListAuditLogs(ctx context.Context, params auditlog.ListAuditLogsParams) middleware.Responder {
+func (a *auditlogAPI) checkPermissionAndBuildQuery(ctx context.Context, qs *string, sort *string, page *int64, pageSize *int64) (*q.Query, error) {
```

Comment:
Do we need to combine the permission check and query building into one function? They seem to be two independent processes

Reply:
In curren context, the query related to the permission, that is why they combined together.